examples/layer_wise_benchmarks/README.md

```
@@ -77,26 +88,61 @@ SLURM_JOB_ID=$SLURM_JOB_ID ./slurm_init_containers.sh
+
+Supported list arguments:
+- `--batch-size` (or `batch_size` in YAML)
+- `--seq-len-q` (or `seq_len_q` in YAML)
+- `--seq-len-kv-cache` (or `seq_len_kv_cache` in YAML)
+- `--balance-ratio` (or `balance_ratio` in YAML)
+
+Command line arguments are comma separated, for example, `--batch-size 1,2,4`. Configs in the YAML file are lists, for example, `batch_size: [1, 2, 4]`.
+
+Run with OpenMPI:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)